src/api/Readme_rubocop.md

```diff
@@ -0,0 +1,11 @@
+# Fixing rubocop styling issues
+
+* Pick one rubocop rule and enable it in .rubocop.yml
+
```

Comment:
But if you pick rule that is already excluding files in the .rubocop_todo.yml and you enable it on .rubocop.yml you don't realize of that files offenses... no? And if you base on that to fix things maybe you left some offenses behind.
So, when you pick the rule from the todo.. better to delete it there, run rubocop, fix and after run the auto-gen-config thing.